app/src/interfaces/_system/system-input-password/input-password.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+withDefaults(
+	defineProps<{
+		value?: string | null;
+		autofocus?: boolean;
+		autocomplete?: string | null;
+		disabled?: boolean;
+	}>(),
+	{
+		value: null,
+		autofocus: false,
+		autocomplete: 'new-password',
+		disabled: false,
+	},
+);
+
+const { t } = useI18n();
+const hidden = ref<boolean>(true);
+
+function toggleHidePassword() {
+	hidden.value = !hidden.value;
+}
+</script>
+
+<template>
+	<v-input
+		:model-value="value"
```

Comment:
The `model-value` binding requires the `@update:modelValue` event in order to propagate the value change upwards.